clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1580,10 +1580,18 @@ protected void checkIngestionProgress(Store store) throws InterruptedException {
   }
 
   protected void updateIngestionRoleIfStoreChanged(Store store) throws InterruptedException {
+    Version version = store.getVersion(versionNumber);
```

Comment:
Can we use version that is already available in SIT instead of looking it up from store

Reply:
After discussed, we could use that version information inside SIT.